util/codec/codec_test.go

```diff
@@ -1176,17 +1227,19 @@ func (s *testCodecSuite) TestHashChunkColumns(c *C) {
 	vecHash := []hash.Hash64{fnv.New64(), fnv.New64(), fnv.New64()}
 	rowHash := []hash.Hash64{fnv.New64(), fnv.New64(), fnv.New64()}
 
-	// Test hash value of the first two `Null` columns
-	for i := 0; i < 2; i++ {
+	// Test hash value of the first 12 `Null` columns
+	for i := 0; i < 12; i++ {
 		c.Assert(chk.GetRow(0).IsNull(i), Equals, true)
 		err1 := HashChunkColumns(sc, vecHash, chk, tps[i], i, buf, hasNull)
-		err2 := HashChunkRow(sc, rowHash[0], chk.GetRow(0), tps, colIdx[i:i+1], buf)
-		err3 := HashChunkRow(sc, rowHash[1], chk.GetRow(1), tps, colIdx[i:i+1], buf)
-		err4 := HashChunkRow(sc, rowHash[2], chk.GetRow(2), tps, colIdx[i:i+1], buf)
+		err2 := HashChunkSelected(sc, vecHash, chk, tps[i], i, buf, hasNull, make([]bool, len(datums)))
```

Comment:
```suggestion
		err2 := HashChunkSelected(sc, vecHash, chk, tps[i], i, buf, hasNull, nil)
```
So all the values are selected ?

Reply:
emmm.... I just want to cover `sel != nil && sel[i] = false` condition, do some modification, maybe better now, ptal if free thx~